app/src/main/java/com/ptpthingers/yacs5e_app/CharacterList.java

```diff
@@ -13,39 +14,49 @@
 public class CharacterList extends Fragment {
 
     private RecyclerView mRecyclerView;
-    private RecyclerView.Adapter mAdapter;
+    private CharacterAdapter mAdapter;
```

Comment:
Maybe change type name to CharacterRecViewAdapterm? It will be easier to maintain code in future.